.changeset/six-pens-look.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rocket.chat/meteor": minor
+---
+
+feat: Setup wizard updates
+
+Updating components and "copy" for more design consistency and removing standalone server (mandatory cloud connectivity)
```

Comment:
changesets should be user-friendly, I suggest rewriting it to something like:

"Setup wizard content updates and enforcing cloud connectivity"

and I would remove everything else